Kudu.Services/Deployment/DeploymentController.cs

```diff
@@ -153,6 +157,22 @@ public void Delete(string id)
                         }
 
                         await _deploymentManager.DeployAsync(repository, changeSet, username, clean, needFileUpdate);
+
+                        // auto-swap
+                        if (_autoSwapHandler.IsAutoSwapEnabled())
+                        {
+                            if (changeSet == null)
+                            {
+                                var targetBranch = _settings.GetBranch();
+                                changeSet = repository.GetChangeSet(targetBranch);
+                            }
+
+                            IDeploymentStatusFile statusFile = _status.Open(changeSet.Id);
```

Comment:
Might consider a `IDeploymentStatusManager.IsSuccessfulChanset` extension method, since same logic in 3 places. But not a big deal.